app/decorators/degree_export_decorator.rb

```diff
@@ -0,0 +1,65 @@
+class DegreeExportDecorator
+  attr_reader :degree
+  NO_DEGREE = 'no degree'.freeze
+
+  def initialize(degree)
+    @degree = degree
+  end
+
+  def qualification_type_hesa_code
+    pad_attribute(:qualification_type_hesa_code, 3)
+  end
+
+  def subject_hesa_code
+    pad_attribute(:subject_hesa_code, 4)
+  end
+
+  def grade_hesa_code
+    pad_attribute(:grade_hesa_code, 2)
+  end
+
+  def institution_country
+    pad_attribute(:institution_country, 2)
+  end
+
+  def institution_hesa_code
+    pad_attribute(:institution_hesa_code, 4)
+  end
+
+  def equivalency_details
+    pad_attribute(:equivalency_details, 2)
+  end
+
+  def start_year
+    year_to_iso8601(fetch_attribute(:start_year))
+  end
+
+  def award_year
+    year_to_iso8601(fetch_attribute(:award_year))
+  end
+
+  def enic_reference
```

Comment:
I may be missing something, is this method being used?

Reply:
Yes, before I was using this decorator to all exports but then I realised is only need for HESA so is a leftover and I deleted now. Good catch!